app/models/custom_button.rb

```diff
@@ -135,7 +135,7 @@ def self.button_classes
   end
 
   def self.name_to_button_class(name)
-    BUTTON_CLASSES.find { |klass| klass.name == name }
+    Rbac.search(:class => name)[0]
```

Comment:
This method needs a test. It should be pretty easy to just stub out Rbac.search since it's already tested